src/environ/_environ_config.py

```diff
@@ -26,6 +26,26 @@
 log = logging.getLogger(CNF_KEY)
 
 
+# We define a sentinel for when prefixes are not set and special handling for
+# what value to use at the app level when that is the case
+class Sentinel(object):
+    def __init__(self, bool_=True):
+        self._bool = bool(bool_)
+
+    def __bool__(self):
+        return self._bool
+
+    __nonzero__ = __bool__  # For Python 2 compatibility
```

Comment:
Could this be protected with an `if PY2`? I'm not a fan to pollute classes with legacy stuff that hopefully nobody will need soon (pls let me dream).

Reply:
As in `six.PY2`, `future.utils.PY2` or a a bespoke constant defined in this package? We'll need to add a dependency or constant definition to do this. What would you prefer?

Reply:
No, this is perfect. We don't need a dependency for a oneliner :)